feature/gribi/encap_frr/README.md

```diff
@@ -428,16 +438,29 @@ the double failure handling, and ensures that the fallback to DEFAULT is
 activated through the backup NHGs of the tunnels instead of withdrawing the
 IPv4Entry.
 
-1.  Update `NHG#8` and `NHG#9` to the following: ``` NHG#8 (Default VRF) {
-    {NH#1000, DEFAULT VRF} } NH#1000 ->
-    { decapsulate_header: OPENCONFIGAFTTYPESDECAPSULATIONHEADERTYPE_IPV4 network_instance: "DEFAULT" }
+1.  Update `NHG#8` and `NHG#9` to the following:
 
-NHG#9 (Default VRF) { {NH#1001, DEFAULT VRF} } NH#1001 -> { decapsulate_header:
-OPENCONFIGAFTTYPESDECAPSULATIONHEADERTYPE_IPV4 network_instance: "DEFAULT" } ```
+    ```
+    NHG#100 (Default VRF) {
```

Comment:
NHG 100, and NHG 101 are pretty munch identical. Is there any reason to keep them separate?

Reply:
they "were" different (encap to different tunnels). To keep things simple, I just do the inline update here.